apps/frontend/src/libs/components/route-card/route-card.tsx

```diff
@@ -0,0 +1,37 @@
+import StarIcon from "./star.svg?react";
+import styles from "./styles.module.css";
+
+type Properties = {
+	alt: string;
+	image?: string;
+	label: string;
+};
+
+const RouteCard = ({ alt, image, label }: Properties): React.JSX.Element => {
+	return (
+		<div className={styles["route-card"]}>
+			{image ? (
+				<img alt={alt} className={styles["image"]} src={image} />
```

Comment:
we won't have alt in db and the data is dynamic (not defined on the frontend)
```suggestion
				<img alt={name} className={styles["image"]} src={image} />
```